apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -525,6 +526,30 @@ async function selectTopStream(
   }
 }
 
+export async function selectTopSql(
+  conn: HasPool,
+  table: string,
+  offset: number,
+  limit: number,
+  orderBy: OrderBy[],
+  filters: TableFilter[] | string,
+  schema = "public",
+  selects = ["*"]
+): Promise<string> {
+  const version = await getVersion(conn)
+  const { query, params } = buildSelectTopQueries({
+    table,
+    offset,
+    limit,
+    orderBy,
+    filters,
+    selects,
+    schema,
+    version,
+  });
+  return knex.raw(query.replaceAll(/\$\d+/g, '?'), params).toQuery();
```

Comment:
Hmm, isn't there a way with knex to have it substitute params?

Reply:
Yeah, I just found that by digging into the repo a little bit. Let me see if I can fix it.

Reply:
I think I misread something. I couldn't find a way to do it with knex. It only let me to use Positional bindings (`?` or `??`) and Named bindings (`:`). I instead copy paste the query builder and hardcode the params.